docs/migration_0.29.md

```diff
@@ -0,0 +1,92 @@
+# Migration Guide from 0.28 to 0.29
+
+OpenTelemetry Rust 0.29 introduces a few breaking changes. This guide aims to
+facilitate a smooth migration for common use cases involving the
+`opentelemetry`, `opentelemetry_sdk`, `opentelemetry-otlp`, and
+`opentelemetry-appender-tracing` crates. For a comprehensive list of changes,
+please refer to the detailed changelog for each crate. This document covers only
+the most common scenarios. Note that changes that only affect custom
+exporter/processor authors are not mentioned in this doc.
+
+OpenTelemetry Metrics API, Log-Bridge API were declared stable in 0.28, and has
+no breaking changes.
+
+## Baggage Changes
+
+The Baggage API has been redesigned to align with the OpenTelemetry
+specification. While the core API for interacting with Baggage remains the same,
+the accepted data types have changed. Baggage Keys now only allow strings (ASCII
+visual characters), and Baggage values are restricted to strings.
```

Comment:
What is visual referring to here? Is it https://www.ascii-code.com/characters/printable-characters?
```suggestion
printable characters), and Baggage values are restricted to strings.
```